discovery-provider/integration_tests/queries/test_search.py

```diff
@@ -235,7 +235,7 @@ def setup_search(db):
         session.execute("REFRESH MATERIALIZED VIEW track_lexeme_dict;")
 
         session.execute(
-            UPDATE_AGGREGATE_USER_QUERY, {"most_recent_indexed_aggregate_block": 0}
+            UPDATE_AGGREGATE_USER_QUERY, {"prev_indexed_aggregate_block": 0}
```

Comment:
just an aside, not a huge fan of the prev_ and current_ convention in aggregates/__init__.py. i think we should pass in a dict with explicit prev and current properties